docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -4,33 +4,53 @@ description = "Enable Grafana 8 Alerts"
 weight = 128
 +++
 
-# Enable Grafana 8 Alerts
+# Opt-in to Grafana 8 alerts
```

Comment:
```suggestion
# Opt-in to Grafana 8 Unified Alerting
```

Reply:
@achatterjee-grafana what do you think about this suggestion?